Android.Jenkinsfile

```diff
@@ -0,0 +1,148 @@
+pipeline {
+	environment {
+		NODE_PATH="/opt/node-v16.3.0-linux-x64/bin"
+		VERSION = sh(returnStdout: true, script: "${NODE_PATH}/node -p -e \"require('./package.json').version\" | tr -d \"\n\"")
+		APK_SIGN_STORE = '/opt/android-keystore/android.jks'
+		APK_SIGN_STORE_PASS = credentials('apk-sign-store-pass')
+		PATH="${env.NODE_PATH}:${env.PATH}"
+		ANDROID_SDK_ROOT="/opt/android-sdk-linux"
+		ANDROID_HOME="/opt/android-sdk-linux"
+	}
+
+	agent {
+		label 'linux'
+	}
+
+	parameters {
+		booleanParam(name: 'PROD', defaultValue: false, description: 'Build for production')
+		booleanParam(
+			name: 'PUBLISH', defaultValue: false,
+			description: "Publish the app to Nexus and Google Play (when not in production mode, " +
+					     "it will publish to the internal testing track of the tutanota-test app on Play)"
+		 )
+	}
+
+	stages {
+		stage('Run Android tests') {
+			steps {
+				dir("${WORKSPACE}/app-android/") {
+					sh "JAVA_HOME=/opt/jdk1.8.0_112 ./gradlew test"
```

Comment:
I don't think we care too much about JDK version? I might be wrong tho

Reply:
we do, the client gets built for 1.8 but by default our environment runs using java 15, and the tests can't run.

Reply:
Java 15 can run Java 1.8 bytecode so I don't see why this is a problem but we can check it later maybe